src/test/resources/com/sap/oss/phosphor/fosstars/model/score/oss/VulnerabilityDiscoveryAndSecurityTestingScoreTestVectors.yml

```diff
@@ -1,5 +1,11 @@
 ---
-defaults: []
+defaults:
+- type: "BooleanValue"
+  feature:
+    type: "BooleanFeature"
+    name: "If a project uses Snyk"
+  flag: false
```

Comment:
Maybe this default value is not required, as you are already putting in unknown values